### PR TITLE
Add `vite-plugin-rune` package and vite template

### PR DESCRIPTION
**Pull request by oblador**

### CODE REVIEW

File: multiplayer/vite-template/src/logic.ts
Comment (helios1138) [inline multiplayer/vite-template/src/logic.ts:1]: Can we add some examples of shared imports here like we discussed? E.g. some lib function that is imported both in the logic.ts and also somewhere in the client.

Comment (oblador) [reply multiplayer/vite-template/src/logic.ts:1]: I don't want to do that in the template, bur rather as a test of the plugin itself.

Comment (oblador) [reply multiplayer/vite-template/src/logic.ts:1]: @helios1138 I've added some integration tests now 👍

**Review by helios1138 — COMMENTED** [review]
Have a couple of questions:
- I tried to test it, but I get a bunch of type errors everywhere. I'm guessing it has to do with multiple "typescripts" inside the monorepo. Not sure if we can do something about it.
- `yarn dev` doesn't start the game for me (`caught (in promise) Error: sdk init failed`)
- why does `main.tsx` imports `App.tsx` as `"./App.tsx"`, but for example `App.tsx` imports `logic.ts` as just `"./logic"`?
- why does `index.html` not include the SDK script?
- why does `index.html` include `logic.ts` script?
- should `vite-template` go under `multiplayer/examples` rather than `multiplayer/`?

**Review by helios1138 — APPROVED** [review]

**Pull request merged**